docs/compliance/refresh_elicensing_data.md

```diff
@@ -25,12 +25,12 @@ Example (get payments for an obligation)
 ```python
```

Comment:
These docs need an update now that we're using the endpoint instead of the flags.